### ISSUE TITLE
Add support for FSA error and meta fields

### ISSUE THREAD

**Issue by cyberixae** [PlatziDev/redux-duck#5]
There is currently no way to set FSA error and meta fields.
Maybe the actionType functions could have another optional argument with the information.
```
actionType(errorPayload, { error: true });
actionType(payload, { meta: { foo: 123 } });
```
This is of course a bit tricky when there is no payload but perhaps it is not too much of a problem
```
actionType(undefined, { error: true });
actionType(undefined, { meta: { foo: 123 } });
```

**Comment by sergiodxa** [comment]
Would you mind moving this issue to https://github.com/sergiodxa/redux-duck/issues?

I recently recovered the package and started maintaining it again since @PlatziDev was not doing it and I would like to work on this.

**Comment by sergiodxa** [comment]
This is published in v1.2.0, feel free to close it here.
